dockerfile/rust/Dockerfile

```diff
@@ -45,7 +45,7 @@ ARG GITHUB_REPO
 ARG VERSION
 ARG BUILD_TIMESTAMP
 
-RUN git clone -b ${VERSION} --single-branch https://${REPO_HOST}/${GITHUB_ORGANIZATION}/${GITHUB_REPO}.git
+RUN git clone -b ${VERSION} --single-branch https://${REPO_HOST}/${GITHUB_ORGANIZATION}/${GITHUB_REPO}.git --recursive
```

Comment:
this will make all rust apps to do a recursive pull.  Is that what we want?

Reply:
yes. If a repo does not have submodules, this is a noop. We want to make sure all build deps are cloned. These steps are done in the build repo, so no storage concerns in the final image.